test/github_test.rb

```diff
@@ -7,5 +7,9 @@ def test_client_being_configurable
 
       assert_equal "ACCESS TOKEN", Github.client.access_token
     end
+
+    def test_access_token_defaulting_to_env_value
+      refute Github.client.access_token.nil?, "Expected `access_token` to get set from OCTOKIT_ACCESS_TOKEN"
```

Comment:
should we test the actual value here?

Reply:
👍 good idea

Reply:
https://github.com/balvig/capistrano-fiesta/commit/ba167caaf84856ef7da9c81b5b2ec12fada58db6